HerPortal.ManagementShell/DatabaseOperation.cs

```diff
@@ -78,10 +93,45 @@ public void RemoveLasFromUser(User user, List<LocalAuthority> lasToRemove)
         using var dbContextTransaction = dbContext.Database.BeginTransaction();
         try
         {
-            foreach (var la in lasToRemove)
-            {
-                user?.LocalAuthorities.Remove(la);
-            }
+            foreach (var la in lasToRemove) user?.LocalAuthorities.Remove(la);
+
+            dbContext.SaveChanges();
+            outputProvider.Output("Operation successful");
+            dbContextTransaction.Commit();
+        }
+        catch (Exception e)
+        {
+            outputProvider.Output($"Rollback following error in transaction: {e.InnerException?.Message}");
+            dbContextTransaction.Rollback();
+        }
+    }
+
+    public void AddConsortiaAndRemoveLasFromUser(User user, List<Consortium> consortia,
+        List<LocalAuthority> localAuthorities)
+    {
+        using var dbContextTransaction = dbContext.Database.BeginTransaction();
```

Comment:
Seems like there's a handful of methods doing the same transaction-wrapping and logging here. Can we do anything to commonise?